### PR TITLE
chore: updating README

### LINKED ISSUE #177: Add Docs to Runtime
### Is your feature request related to a problem? Please describe
I want to know more about uds runtime but i don't have any docs

### Describe the solution you'd like
docs that will get added to the uds docs like cli / core

### PR DESCRIPTION
**Pull request by TristanHoladay**
## Description
Add docs showing how to deploy and run the application

## Related Issue

resolves #177

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,103 @@
 # UDS Runtime
 
-## Development
+UDS Runtime is a cluster analytics and insights web application, capable of running locally or in cluster.
```
Comment (UncleGedd) [inline README.md:3]: "analytics" is a trigger word for me, makes me think of data analytics which we aren't doing (yet!)

```suggestion
UDS Runtime is the frontend for all things UDS, providing views and insights into your UDS cluster.
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,103 @@
...
 
-To start the backend server, run the following command:
+## Quickstart Deploy
 
+> !**WARNING**  
+> DO NOT USE IN PRODUCTION ENVIRONMENTS AT THIS TIME
```
Comment (UncleGedd) [inline README.md:8]:
```suggestion
> This product is not recommended for production use at this time
```

Comment (UncleGedd) [inline README.md:8]:
```suggestion
> !**WARNING**  
> UDS Runtime is in early alpha, expect breaking changes to functionality 
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,103 @@
...
+
+### Pre-requisites
+
+Recommended:
+* [uds-cli](https://github.com/defenseunicorns/uds-cli#install)
```
Comment (UncleGedd) [inline README.md:13]:
```suggestion
* [UDS-CLI](https://github.com/defenseunicorns/uds-cli#install)
```

File: README.md [hunk 4]
```diff
@@ -1,19 +1,103 @@
...
+
+If building locally:
+* `Go >= 1.22.0`
+* `Node >= v21.1.0`
+
+### In Cluster
+
+Assumes cluster is running and `uds-core` is deployed.
```
Comment (UncleGedd) [inline README.md:21]:
```suggestion
Assumes a K8s cluster is running.
```

**Comment by BillyFigueroa** [comment]
Should the teach stack include the charting library we are using? That will likely change but how do we decide what to add to the front end stack? oh, also I think we should add Typescript

File: README.md [hunk 5]
```diff
@@ -1,19 +1,103 @@
...
+
+#### Standalone Package
+```bash
+uds deploy ghcr.io/defenseunicorns/packages/uds/uds-runtime:<tag> --confirm
+```
+
+#### In Bundle
```
Comment (UncleGedd) [inline README.md:28]: IMO this example is a bit extra for now, but I like the idea of using an example bundle as a smoke test and pointing to it from the docs. Personally, I'd remove from here and add to https://github.com/defenseunicorns/uds-runtime/issues/152

File: README.md [hunk 6]
```diff
@@ -1,19 +1,103 @@
...
 
-To start the frontend server, run the following command:
+### To start the frontend server, run the following command:
+
+**With uds-cli**
+```bash
+uds run dev-ui
+```
+
+**Wihtout uds-cli**
```
Comment (UncleGedd) [inline README.md:88]:
```suggestion
**Without UDS-CLI**
```

File: README.md [hunk 7]
```diff
@@ -1,19 +1,103 @@
...
 ```bash
 air
 ```
 
 > **NOTE**: If you do not have air installed, you can find instructions for how to install at [here](https://github.com/air-verse/air)
 
-To start the frontend server, run the following command:
+### To start the frontend server, run the following command:
+
+**With uds-cli**
```
Comment (UncleGedd) [inline README.md:83]:
```suggestion
**With UDS-CLI**
```

File: README.md [hunk 8]
```diff
@@ -1,19 +1,103 @@
...
+1. compile: `uds run compile`
+1. run: `./build/main`
+
+## Quickstart Development
+
+For a full guide on developing for UDS Runtime, please read the [CONTRIBUTING.md](./CONTRIBUTING.md)
+
+### To start the backend development server, run the following command:
+
+**With uds-cli**
```
Comment (UncleGedd) [inline README.md:69]:
```suggestion
**With UDS-CLI**
```

File: README.md [hunk 9]
```diff
@@ -1,19 +1,103 @@
...
+```bash
+uds run dev-server
+```
+
+**Without uds-cli**
```
Comment (UncleGedd) [inline README.md:74]:
```suggestion
**Without UDS-CLI**
```

**Comment by TristanHoladay** [comment]
@BillyFigueroa good catch on adding typescript. i went ahead and added svelte-apexcharts as well for now. we can update when that changes.

File: README.md [hunk 10]
```diff
@@ -1,19 +1,75 @@
...
+## Nightly Releases
+
+UDS Runtime publishes a canary release of latest changes every night tagged `nightly-unstable`
 
 ```bash
-cd ui && npm run dev
+uds deploy ghcr.io/defenseunicorns/packages/uds/uds-runtime:nightly-unstable
 ```
 
 ## Techstack
```
Comment (UncleGedd) [inline README.md:75]:
```suggestion
## Tech Stack
```

**Review by UncleGedd — APPROVED** [review]

**Pull request merged**